timemachine/cpp/src/kernels/k_exchange.cu

```diff
@@ -327,16 +263,11 @@ void __global__ k_adjust_weights(
             // A loop that in the case of water will be 3x3
             for (int i = idx_in_batch * mol_size; i < (idx_in_batch + 1) * mol_size; i++) {
                 for (int j = min_atom_idx; j <= max_atom_idx; j++) {
-                    weight_accumulator += FLOAT_TO_FIXED_ENERGY<RealType>(inv_kT * per_atom_energies[i * N + j]);
+                    weight_accumulator += FLOAT_TO_FIXED_ENERGY<RealType>(per_atom_energies[i * N + j]);
```

Comment:
This now only accumulates weights and the `k_convert_energies_to_log_weights` does the converstion to unitless quantities. This was part of the move to storing everything in fixed point. If we add back in the conversion here it would break the bitwise determinism between a proposal per move of 1 and proposal per move of >1